test/e2e/tests/test_hpo.py

```diff
@@ -18,40 +18,56 @@
 import logging
 from typing import Dict
 
-from e2e import service_marker, create_sagemaker_resource
-from e2e.replacement_values import REPLACEMENT_VALUES
-from acktest.resources import random_suffix_name
+from acktest.resources import random_suffix_name, load_resource_file
 from acktest.k8s import resource as k8s
+from e2e import (
+    resource_directory,
+    CRD_GROUP,
+    CRD_VERSION,
+    service_marker,
+    create_sagemaker_resource,
+    wait_for_status,
+)
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.bootstrap_resources import get_bootstrap_resources
+from time import sleep
 
 RESOURCE_PLURAL = "hyperparametertuningjobs"
-HPO_JOB_STATUS_CREATED = ("InProgress", "Completed")
-HPO_JOB_STATUS_STOPPED = ("Stopped", "Stopping")
```

Comment:
Any reason you moved these out of module scope and into class scope of the `TestHPO` class?

Reply:
Seemed cleaner to me this way but on second thought, I could actually move all of these out to a shared config since the values of these statuses are the same across job style resources.

Reply:
+1 to move to common file